docs/devel/testing.md

```diff
@@ -82,13 +82,13 @@ passing, so it is often a good idea to make sure the e2e tests work as well.
 
 ### Run all unit tests
 
-The `hack/test-go.sh` script is the entrypoint for running the unit tests that
-ensures that `GOPATH` is set up correctly.  If you have `GOPATH` set up
-correctly, you can also just use `go test` directly.
+`make test` is the entrypoint for running the unit tests that ensures that
+`GOPATH` is set up correctly.  If you have `GOPATH` set up correctly, you can
```

Comment:
does `make test` build autogenerated files (it needs to, or test will not be testing the right thing)? if so, the last sentence is probably not true.